openshift/4.0/templates/Logging/role-binding.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: pod-reader
+  namespace: 3cd915-dev
```

Comment:
You may want to parameterize this configuration, as currently it's only good for DEV

Reply:
@Fosol You can run this locally by just running docker-compose up logging from the /Logging dir
The build context is already there to point docker-compose.yml to the dockerfile. See the build: Context: . in the docker-compose file.

I will update the readme and parameter the dc to apply to other env